website/util/__init__.py

```diff
@@ -123,7 +123,7 @@ def api_v2_url(path_str,
     return str(base_url)
 
 
-def web_url_for(view_name, _absolute=False, _guid=False, *args, **kwargs):
+def web_url_for(view_name, _absolute=False, _internal=False, _guid=False, *args, **kwargs):
```

Comment:
`osf/models/node.py:    def web_url_for`  and
`website/project/model.py:    def web_url_for`  call
`website/util/__init__.py:def web_url_for` with kwargs so I think we're good here.